local/sdk-sriov/pkg/k8s/manager.go

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package k8s provides tools to access k8s API
+package k8s
+
+import (
+	"context"
+	"path"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/google/uuid"
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+	podresources "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+
+	"github.com/networkservicemesh/cmd-forwarder-sriov/local/sdk-sriov/pkg/tools"
+)
+
+const (
+	dialTimeoutDefault = 15 * time.Second
+	kubeletSocket      = "kubelet.sock"
+)
+
+// Manager is a helper class to work with the k8s API
+type Manager interface {
```

Comment:
Why do we need this interface?

Reply:
OK, now there is a public struct and this interface moved to 'server' for the better testing.